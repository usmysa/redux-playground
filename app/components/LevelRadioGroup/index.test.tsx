import { renderWithProviders } from "@/__tests__";

import { LevelRadioGroup } from ".";

describe("LevelRadioGroup", () => {
  it("renders correctly", () => {
    const { container } = renderWithProviders(<LevelRadioGroup />);

    expect(container).toMatchSnapshot();
  });
});
