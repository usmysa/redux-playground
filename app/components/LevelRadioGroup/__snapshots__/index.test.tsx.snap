// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LevelRadioGroup > renders correctly 1`] = `
<div>
  <form>
    <div
      class="styles_wrapper__1xqviej9"
      style="--styleVars_alignItems__1xqviej0: normal; --styleVars_flexDirection__1xqviej1: column; --styleVars_gap__1xqviej2: var(--spacing-xs__xzbfpk1); --styleVars_justifyContent__1xqviej4: flex-start; --styleVars_marginTop__1xqviej5: 0px; --styleVars_padding__1xqviej6: 0px;"
    >
      <div
        class="styles_wrapper__1xqviej9"
        style="--styleVars_alignItems__1xqviej0: normal; --styleVars_flexDirection__1xqviej1: row; --styleVars_gap__1xqviej2: var(--spacing-xs__xzbfpk1); --styleVars_justifyContent__1xqviej4: flex-start; --styleVars_marginTop__1xqviej5: 0px; --styleVars_padding__1xqviej6: 0px;"
      >
        <input
          aria-label="Easy"
          class="styles_radio__tp90la1"
          id="level-Easy"
          name="level"
          type="radio"
          value="Easy"
        />
        <label
          class="styles_labelStyle__tp90la0"
          for="level-Easy"
        >
          Easy
        </label>
      </div>
      <div
        class="styles_wrapper__1xqviej9"
        style="--styleVars_alignItems__1xqviej0: normal; --styleVars_flexDirection__1xqviej1: row; --styleVars_gap__1xqviej2: var(--spacing-xs__xzbfpk1); --styleVars_justifyContent__1xqviej4: flex-start; --styleVars_marginTop__1xqviej5: 0px; --styleVars_padding__1xqviej6: 0px;"
      >
        <input
          aria-label="Normal"
          class="styles_radio__tp90la1"
          id="level-Normal"
          name="level"
          type="radio"
          value="Normal"
        />
        <label
          class="styles_labelStyle__tp90la0"
          for="level-Normal"
        >
          Normal
        </label>
      </div>
      <div
        class="styles_wrapper__1xqviej9"
        style="--styleVars_alignItems__1xqviej0: normal; --styleVars_flexDirection__1xqviej1: row; --styleVars_gap__1xqviej2: var(--spacing-xs__xzbfpk1); --styleVars_justifyContent__1xqviej4: flex-start; --styleVars_marginTop__1xqviej5: 0px; --styleVars_padding__1xqviej6: 0px;"
      >
        <input
          aria-label="Hard"
          class="styles_radio__tp90la1"
          id="level-Hard"
          name="level"
          type="radio"
          value="Hard"
        />
        <label
          class="styles_labelStyle__tp90la0"
          for="level-Hard"
        >
          Hard
        </label>
      </div>
    </div>
  </form>
</div>
`;
